common/src/calendar/util.ts

```diff
@@ -151,11 +151,12 @@ export const isSlotWithinTimeframe = (
 };
 
 const doesSlotCollideWithApplicationRounds = (
-  applicationRounds: ApplicationRound[] | ApplicationRoundType[] = [],
-  slot: Date
+  slot: Date,
+  applicationRounds: ApplicationRound[] | ApplicationRoundType[] = []
```

Comment:
What's the motivation to switch the order?

Reply:
If i remember correctly , there is a common rule (also enforced by modern Typescript) that function variables with default values should be after required variables. Same way as optional variables.